core/src/main/java/org/kohsuke/stapler/MetaClassLoader.java

```diff
@@ -62,6 +64,7 @@ public static MetaClassLoader get(ClassLoader cl) {
     /**
      * If non-null, delegate to this classloader.
      */
+    @SuppressFBWarnings(value = "MS_SHOULD_BE_REFACTORED_TO_BE_FINAL", justification = "Legacy switch.")
```

Comment:
OK, I thought it was non`final` merely because the author had not bothered to factor the initialization expression into a method, which would then be illegal acc. to JLS even though it is not mutated after class initialization.